docs/onboard-to-seed.md

```diff
@@ -0,0 +1,50 @@
+# Onboard to SEED
+
+Currently Public Officers can onboard their device to SEED by logging into the TechPass portal using non-SE GSIB machine.
+
+<div class="warn">
+ <ul>
+ <li>SEED is applicable only for <b>non-GSIB device</b>.</li>
+ <li>If you are a vendor or contractor who needs to be onboarded to SEED, contact your project manager or reporting officer.</li>
+ <li>Click <a href="https://docs.developer.tech.gov.sg/docs/security-suite-for-engineering-endpoint-devices">here</a> for more information about SEED.</li>
+ </ul>
+</div>
+
+_To onboard your device to SEED:_
```

Comment:
are these steps telling how to onboard a device to SEED or is it to tell how to get a SEED licence?